index.html

```diff
@@ -111,98 +276,294 @@ <h1>
-      <details>
-        <summary>Example Tag</summary>
-        <img src="./img/ua_legacy_config.png">
-      </details>
-    </div>
-    <textarea id="ua-legacy-result" readonly cols="60">
-Press Submit to generate data.
-    </textarea>
-  </div>
-  <div>
-    <h2>
-      Schema Conversion
-    </h2>
-    <div>
-      To send this data using the gtag Google Analytics 4 schema, use the following command:
-    </div>
-    <textarea id="ga4-gtag-command" readonly cols="60">
-Press Submit to generate data.
+    -->
+    <button id="submit">Analyze</button>
+  </section>
+
+
+  <h2 id="analysis">Results of analysis</h2>
+  <section>
+    <h3>Identification</h3>
+    <section>
+      <p>Identify the library and version of Google Analytics for the event
+        above.</p>
+      <div id="schema-not-submitted">Pending analysis...</div>
+      <div class="hidden" id="schema-known">
+        This appears to be a
+        <a class="hidden"
+          href="https://developers.google.com/tag-manager/enhanced-ecommerce"
+          id="schema-ua-gtm">Universal Analytics (Tag Manager) event</a>
+        <a class="hidden"
+          href="https://developers.google.com/tag-manager/ecommerce-ga4"
+          id="schema-ga4-gtm">Google Analytics 4 (Tag Manager) event</a>
+        <a class="hidden"
+          href="https://developers.google.com/analytics/devguides/collection/gtagjs/ecommerce"
+          id="schema-ua-gtag">Universal Analytics (gtag.js) event</a>
+        <a class="hidden"
+          href="https://developers.google.com/analytics/devguides/collection/ga4/ecommerce"
+          id="schema-ga4-gtag">Google Analytics 4 (gtag.js) event</a>
+        <span class="hidden" id="schema-gtag-unknown">
+          <a
+            href="https://developers.google.com/analytics/devguides/collection/gtagjs/ecommerce">Universal
+            Analytics (gtag.js) event</a> <b>OR</b> <a
+            href="https://developers.google.com/analytics/devguides/collection/ga4/ecommerce">Google
+            Analytics 4 (gtag.js) event</a>
+        </span>
+        .
+      </div>
+      <span class="hidden" id="schema-unknown">The data you entered is not
+        recognized as a Universal Analytics or a Google Analytics 4 event.
+      </span>
+    </section>
+
+    <h3>Validation</h3>
+    <section>
+      <p>Use the following information to confirm how the event above is
+        interpreted by various libraries and tag configurations.</p>
+
+      <h4 id="validation-ga4">Google Analytics 4</h4>
+      <p>
+        For the following configurations:</p>
+      <ul>
+        <li>Tag Manager <b>Google Analytics 4 event tag</b> with <b>Send
+            Ecommerce from Data Layer</b> enabled.
+          <ul>
+            <li class="no-bullet">
+              <details>
+                <summary>Show example GA4 event tag configuration</summary>
+                <img src="./ga4_tag_config.png">
+              </details>
```

Comment:
Nit. An unstylized details element is hard to see as expandable. Might want to add  "click to expand" text to this element.